.gitignore

```diff
@@ -1,5 +1,6 @@
 # binary.
 scorecard
+!pkg/scorecard
```

Comment:
Is this intentional?

Reply:
Yes. Otherwise files in `pkg/scorecard` were being ignored because the line above it says don't track anything with the name "scorecard". Which was intended for the binary, but applies to directories as well.